packages/block-library/src/comment-date/index.php

```diff
@@ -31,13 +31,13 @@ function render_block_core_comment_date( $attributes, $content, $block ) {
 	$link               = get_comment_link( $comment );
 
 	if ( ! empty( $attributes['isLink'] ) ) {
-		$formatted_date = sprintf( '<a href="%1s">%2s</a>', esc_url( $link ), esc_html( $formatted_date ) );
+		$formatted_date = sprintf( '<a href="%1s">%2s</a>', esc_url( $link ), $formatted_date );
 	}
 
 	return sprintf(
 		'<div %1$s><time datetime="%2$s">%3$s</time></div>',
 		$wrapper_attributes,
-		esc_attr( get_comment_date( 'c', $comment ) ),
+		get_comment_date( 'c', $comment ),
```

Comment:
Core function - can be trusted, maybe filtered.

Reply:
I think it's okay to escape this value - https://github.com/Automattic/_s/blob/master/inc/template-tags.php#L23

Reply:
Unfortunately as I'm learning, anything that can be filtered can contain HTML. 

https://github.com/WordPress/wordpress-develop/blob/f0180f59ea00683647c602941c1f2b422192df46/src/wp-includes/comment-template.php#L568

So I'm not sure. Most of these "reverts" are on core functions because of this fact.

Reply:
In this case, it should never contain HTML since the value is for the `datetime` attribute.

Reply:
> it should never contain HTML since the value is for the datetime attribute.

That's the key point. Thanks 👍

Reply:
✅